bundles/EcommerceFrameworkBundle/PaymentManager/Payment/Mpay24Seamless.php

```diff
@@ -268,21 +300,28 @@ public function getInitPaymentRedirectUrl($config): array
     private function addOrderItemPositions(OnlineShopOrder $order, string $paymentType, array $additional): array
     {
         $checkSum = 0.0;
-        $checkSumVat = 0.0;
+        $checkSumVat= 0.0;
+
+        $orderTotalPrice = round($order->getTotalPrice(),2);
```

Comment:
shouldn't we calculate here with Decimals? to prevent rounding errors?

Reply:
Yes, that's an improvmeent suggestion.
However, currently the addOrderItemPositions() method is not applied in the official version of the provider, because the standard Mpay24 PHP SDK does not support this feature yet. 
Also see https://github.com/mpay24/mpay24-php/pull/79#issuecomment-383528608

Reply:
I added 
```php
addOrderItemPositions
```
method for completeness, and as it is currently in use by one of our customer projects. Hower, as it is not officially supported by Mpay24 I wouldn't invest more effort into upgrading to decimals.